src/Runner.php

```diff
@@ -43,6 +44,16 @@ class Runner implements ContainerAwareInterface
      */
     protected $selfUpdateRepository = null;
 
+    /**
+     * @var \Composer\Autoload\ClassLoader
+     */
+    protected $classLoader = null;
+
+    /**
+     * @var string
+     */
+    protected $commandFileRelativeNamespace = 'Robo\Commands';
```

Comment:
Remove the auto-initialized constant, and inject it from the Robo script (so folks searching for the default value will find it in only one place).

Perhaps the default should be something like `Robo\Plugin`, so that we will then have `\PLUGIN_NAME\Robo\Plugin\Commands`, \PLUGIN_NAME\Robo\Plugin\EventHandler` or whatever is needed.